src/main/java/com/kobylchak/bookstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.kobylchak.bookstore.service;
+
+import com.kobylchak.bookstore.model.Book;
+import com.kobylchak.bookstore.repository.BookRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class BookServiceImpl implements BookService {
+
+    private BookRepository bookRepository;
+
+    @Autowired
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
```

Comment:
lets use `RequiredArgsConstructor` annotation if we added it
```suggestion
    private final BookRepository bookRepository;
```